packages/components/psammead-radio-schedule/src/index.jsx

```diff
@@ -11,8 +11,14 @@ const StartTimeWrapper = styled.div`
   padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};
 `;
 
-// Using flex-box on browsers that do not support grid will break grid fallback defined in psammead-grid
+// Reset default of <ul> style
 const StyledGrid = styled(Grid)`
+  padding: 0;
+  margin: 0;
+`;
+
+// Using flex-box on browsers that do not support grid will break grid fallback defined in psammead-grid
+const StyledFlexGrid = styled(Grid)`
   @supports (display: grid) {
```

Comment:
You could use here the custom `import { grid } from '@bbc/psammead-styles/detection';` we have in Psammead to detect grid support.